src/config/global.styles.ts

```diff
@@ -1,11 +1,11 @@
-import { createGlobalStyle } from 'styled-components';
-import './fonts/styles.css';
+// import { createGlobalStyle } from 'styled-components';
+// import './fonts/styles.css';
 
-export const GlobalStyles = createGlobalStyle`
-  * {
-    font-family: 'Arizona Serif';
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-  }
-`;
+// export const GlobalStyles = createGlobalStyle`
```

Comment:
And maybe we could clean this as well, if we are not using it in the future, if we will, let's leave it